helper/TestCenterHelper.php

```diff
@@ -0,0 +1,392 @@
+use oat\taoProctoring\model\implementation\DeliveryService;
+use oat\taoProctoring\model\implementation\TestSessionHistoryService;
+use oat\taoProctoring\model\EligibilityService;
+use oat\taoProctoring\model\DeliveryExecutionStateService;
+use oat\taoProctoring\model\PaginatedStorage;
+use oat\taoProctoring\model\deliveryLog\DeliveryLog;
+use oat\taoTestCenter\helper\BreadcrumbsHelper;
+use oat\taoTestCenter\model\TestCenterService;
+use oat\taoProctoring\helpers\DataTableHelper;
+
+/**
+ * This temporary helpers is a temporary way to return data to the controller.
+ * This helps isolating the mock code from the real controller one.
+ * It will be replaced by a real service afterward.
+ */
+class TestCenterHelper
+{
+    /**
+     * Gets a list of available test sites
+     *
+     * @param array [$options]
+     * @return array
+     * @throws ServiceNotFoundException
+     * @throws \common_Exception
+     * @throws \common_exception_Error
+     */
+    public static function getTestCenters($options = array())
+    {
+        $testCenterService = TestCenterService::singleton();
+        $currentUser = \common_session_SessionManager::getSession()->getUser();
+
+        $testCenters = $testCenterService->getTestCentersByProctor($currentUser, $options);
+        $entries = array();
+        foreach ($testCenters as $testCenter) {
+            $entries[] = array(
+                'id' => $testCenter->getUri(),
+                'url' => _url('testCenter', 'TestCenter', null, array('testCenter' => $testCenter->getUri())),
+                'label' => $testCenter->getLabel(),
+                'text' => __('Go to')
+            );
+
+        }
+        return $entries;
+    }
+
+    /**
+     * Gets a list of available test sites
+     *
+     * @param string $testCenterId
+     * @return core_kernel_classes_Resource
+     * @throws ServiceNotFoundException
+     * @throws \common_Exception
+     * @throws \common_exception_Error
+     */
+    public static function getTestCenter($testCenterId)
+    {
+        $testCenterService = TestCenterService::singleton();
+
+        return $testCenterService->getTestCenter($testCenterId);
+    }
+
+    /**
+     * Gets a list of entries available for a test site
+     *
+     * @param $testCenter core_kernel_classes_Resource
+     * @return array
+     * @throws ServiceNotFoundException
+     * @throws \common_Exception
+     * @throws \common_exception_Error
+     */
+    public static function getTestCenterActions(core_kernel_classes_Resource $testCenter)
+    {
+
+        $actionDiagnostics = BreadcrumbsHelper::diagnostics($testCenter);
+        $actionDeliveries = BreadcrumbsHelper::deliveries($testCenter);
+
+        $actions = array(
+            array(
+                'url' => $actionDeliveries['url'],
+                'label' => __('Sessions'),
+                'content' => __('Monitor and manage sessions for the test site'),
+                'text' => __('Go')
+            )/*,
+            array(
```

Comment:
readiness check is being refactored as a separate task and can be reintegrated after this work is completed